datausage.qmd

```diff
@@ -0,0 +1,41 @@
+---
+title: "U.S. Greenhouse Gas Center: Data Usage Notebooks"
+---
+
+Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) data usage notebooks, your gateway to exploring and analyzing curated datasets on greenhouse gas emissions.
+Our cloud-based system offers seamless access to GHG curated datasets. Dive into the data with our data usage Jupyter notebooks, which demonstrate how to explore, access, visualize, and conduct basic data analysis for each GHG Center dataset in a code notebook environment. The data usage notebooks are grouped topically. The dataset product type (model output, satellite observation, etc.) is noted next to the notebook name. Click on a notebook to learn more about the dataset and to view the data usage code.
```

Comment:
Recommend removing the product type from next to the notebook names and remove sentence "The dataset product type (model output, satellite observation, etc.) is noted next to the notebook name."